### ISSUE TITLE
Large  json content with more than 4096 items having 100k failure

### ISSUE THREAD

**Issue by comarius** [miloyip/rapidjson#995]
I have a pretty large json as a result over a query. The 
doc.Parse(json).HasParseError() yields no errors, but when I am trying to access 
the doc is asserting on all iS****()  tests. 
The json is an array of 4096 items repeating pretty much this sequence, and has close to 100K
[...{"T":"a text which describes the label T"},...]
Any ideas ? 
Thank you.

**Comment by miloyip** [comment]
RapidJSON can handle jsons in GBs upon enough memory.
One default limit is 2^32 - 1 for array/object size, which can be extended to 2^64 on 64-bit platform.
Can you show the relevant code, and even better with a sample JSON uploaded somewhere?

**Comment by comarius** [comment]
Hi,
Thank you . I figure it out.I increase the 64*1024 chun-ksize in allocator, and works 'RAPID'
Thank you very much.

**Comment by miloyip** [comment]
If this change works, then there should be a bug...

**Comment by StilesCrisis** [comment]
Yes, please submit your code and JSON file so we can investigate.
